serializr.js

```diff
@@ -167,8 +167,30 @@
             }
         }
 
+				// Ugly way to get the parameter names since they aren't easily retrievable via reflection
+				const STRIP_COMMENTS = /((\/\/.*$)|(\/\*[\s\S]*?\*\/))/mg;
+				const ARGUMENT_NAMES = /([^\s,]+)/g;
+				function getParamNames(func) {
```

Comment:
minimization will break this mechanism, probably it should be mandatory, or at least warned, to use alias if constructor arguments are decorated so that the JSON name is not lost? E.g. `constructor(@serializable public name:string)` might be minimized to `constructor(@serializable public a:string)`, to avoid breaking the schema it should be `constructor(@serializable(alias("name", true)) public a)`.

But maybe it is not a problem at all as minification usually happens after compilation, which probably turns this into a member assignment?